cljfmt/test/cljfmt/assert/cljs.cljc

```diff
@@ -0,0 +1,10 @@
+(ns cljfmt.assert.cljs
+  (:require [cljs.test :as test]
+            [cljfmt.assert.impl :as impl]))
+
+;; cljs macros are compiled by clj
+#?(:clj
```

Comment:
Why do you have `cljfmt.assert.cljs`, `cljfmt.assert.impl` and `cljfmt.asserts`? Can you put everything into `cljfmt.core-test`?

Reply:
My code was heavily inspired by [this example](https://github.com/grzm/ex.assertions) which splits things out in a very similar way. I personally prefer to move self contained complexity out to separate files.  I find when source files get larger they also get harder to digest and maintain. I realize this is a subjective call and am happy to move to your preference.

`asserts.cljc` could definitely be merged into `core_test.cljc`, but I'm thinking there might be some file separation gymnastics needed to support test assertions under ClojureScript: `assert/cljs.cljc` is required for ClojureScript but is compiled by Clojure. I am new to this area though, I'll try to understand what is possible and get back to you.

You did not ask but: I added the `assert` sub-directory to try to convey these are internal details that `core_test` need not concern itself with directly.

Reply:
I'd rather just move the method into `core_test.cljc`, or possibly `test_util.cljc`. The way the cljs macros interact might make placing it in a secondary namespace a good idea. However, the namespace should indicate clearly that it's a test namespace, hence `test-util`. You should also be able to do it with only one `.cljc` file.

Reply:
I have not succeeding at merging everything into `core_test.cljc`. I did manage to merge `asserts.cljc` into `core_test.cljc`.  I'm left with the cljs `assert-expr` in a separate source file, and the code that the clj and cljs `assert-expr`s share in another file.  My last push called these `assert_cljs.cljc` and `assert_impl.cljc` respectively.  Do you think moving these two files under a `test-util` subdir would work for you?

BTW, thanks for your patience and encouragement, it is much appreciated.

Reply:
Thanks for clearing that up, I was thinking I was missing some concept or other.  

If I understand correctly, which I might not, `common` and `cljs` files need the `cljc` extension. 

Your new separation makes usage from ns in `core-tests` very clear.  I also like your tweaks to the code, you have a real talent for succinctness.

Note: I've left the order of args when including options as: `in-line options expected-lines`. I feel they belong next to `in-lines` for better readability of tests.

Reply:
> If I understand correctly, which I might not, `common` and `cljs` files need the `cljc` extension.

What's your reasoning? If `assert-expr` is used in `is`, and `is` is a macro, doesn't that mean that only Clojure files are necessary?

Reply:
I misled you, I don't understand yet. Let me rephrase: I do not know why, but to get the build to succeed, I needed to rename `common` and `cljs` files with the `cljc` extension.

Reply:
Okay, lets set them as `cljc` then.

Reply:
Ok, I've pushed changes, they are ready for your review.